test/uk/gov/hmrc/fileupload/infrastructure/PlayHttpSpec.scala

```diff
@@ -1,132 +0,0 @@
-/*
- * Copyright 2017 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package uk.gov.hmrc.fileupload.infrastructure
-
-import cats.data.Xor
-import com.github.tomakehurst.wiremock.WireMockServer
-import com.github.tomakehurst.wiremock.client.ResponseDefinitionBuilder
-import com.github.tomakehurst.wiremock.client.WireMock._
-import com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig
-import com.github.tomakehurst.wiremock.http.Fault
-import org.scalatest.concurrent.Eventually
-import org.scalatest.time.{Seconds, Span}
-import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach, Suite}
-import play.api.http.Status
-import play.api.libs.json.{JsSuccess, Json}
-import play.api.libs.ws.WSRequest
-import uk.gov.hmrc.fileupload.TestApplicationComponents
-import uk.gov.hmrc.fileupload.infrastructure.PlayHttp.PlayHttpError
-import uk.gov.hmrc.fileupload.transfer.FakeAuditer
-import uk.gov.hmrc.play.audit.http.config.{AuditingConfig, BaseUri, Consumer, LoadAuditingConfig}
-import uk.gov.hmrc.play.audit.http.connector.AuditConnector
-import uk.gov.hmrc.play.config.RunMode
-import uk.gov.hmrc.play.http.HeaderCarrier
-import uk.gov.hmrc.play.test.UnitSpec
-
-import scala.collection.mutable.ListBuffer
-import scala.concurrent.ExecutionContext.Implicits.global
-
-class PlayHttpSpec extends UnitSpec with BeforeAndAfterEach with TestApplicationComponents with Eventually with FakeAuditer {
```

Comment:
Why is it deleted?

Reply:
Removed it as the only thing it covers is auditing, and we don't use auditing (it's disabled in all environments)

Reply:
Just in case you not have noticed that: This branch has conflicts with master now